ang/civicase/Search.js

```diff
@@ -83,22 +83,8 @@
      * Watcher for expanded state and update tableHeader top offset likewise
      */
     function expandedWatcher () {
-      $timeout(function () {
-        var bodyPadding = parseInt($('body').css('padding-top'), 10); // to see the space for fixed menus
-        var $tableHeader = $('.civicase__case-list-table__header');
-        var $caseNavigation = $('.civicase__case-body_tab');
-        var $casePanelBody = $('.civicase__case-details-panel > .panel-body');
-
-        // Reset left case tab header
-        if ($tableHeader.data('bs.affix')) {
-          $tableHeader.data('bs.affix').options.offset.top = $tableHeader.offset().top - bodyPadding;
-        }
-
-        // Resert right case view tab header
-        if ($caseNavigation.data('bs.affix')) {
-          $caseNavigation.data('bs.affix').options.offset.top = $casePanelBody.offset().top - bodyPadding;
-        }
-      });
+      $rootScope.$broadcast('stickyCaseListHeaderReinitiatePos');
```

Comment:
Use `$emit`

Reply:
We dont need two events. Just fire a single event to denote, UI has changed. That should be listened by all.
For example, `civicase::case-list::header-position-changed`

Reply:
> Use `$emit`

Limitation of using $emit is that it can be used in hierarchical way (i.e. any emitted event can only be caught by its parent scope or itself) but this event is being broadcasted from multiple controllers (Search.js) and (Bulkactionmessage.js) then we must need to use rootscope broadcast.

Reply:
Lets discuss this.